protocol.py

```diff
@@ -0,0 +1,29 @@
+from LSP.plugin.core.protocol import Location, Position, RangeLsp, TextDocumentIdentifier
```

Comment:
Just a heads up, if someone opens an issue that `LSP-typescript: Find callers` is not available in the command palette, ask them to see if they are running the latest LSP.

I didn't had the latest version of LSP, and there was an error in the console stating that it couldn't find `TextDocumentIdentifier`.

Reply:
When PC updates this package to the latest version that includes this code then it should also be guaranteed that LSP is at the latest version.

Unless it's an ST3, that is. So I guess I can add some safe guard for that.

Reply:
> This code will throw exception in ST3 but normal functionality will still work, only that command won't be available.

Yes and that is fine by me :)